svelte.config.js

```diff
@@ -0,0 +1,31 @@
+import preprocess from 'svelte-preprocess';
+import { mdsvex } from 'mdsvex';
+
+const extensions = [`.svelte`, '.md', `.mdx`, '.svx'];
+
+/** @type {import('@sveltejs/kit').Config} */
+const config = {
+	preprocess: [
+		preprocess(),
+		mdsvex({
+			extensions: extensions,
+			layout: { eventPage: './src/lib/layouts/EventPage.svelte' }
+		})
+	],
+	extensions: extensions,
+	kit: {
+		// hydrate the <div id="svelte"> element in src/app.html
+		target: '#svelte',
```

Comment:
I think this is the only option you need in the `kit` config. The rest below this all look like defaults that can be removed

Reply:
Ok, I'll try when I finished the endpoint 👍

Reply:
23f66bf should resolve that